### PR TITLE
Fix instructions for local Windows build

### PR DESCRIPTION
**Pull request by NatasaKralj**
New instructions and command to run a local build on Windows.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,9 +51,24 @@ Node.js is a JavaScript runtime. You can install it by downloading the installer
 
 #### Running the Server
 
+##### Windows
+
+The steps in this section assume you have installed Hugo and Node in the root directory of your local repository. For example, Hugo is installed in C:\Projects\Mendix\docs\hugo. If this is not the case, you will have to edit the commands in step 3 to reflect their install paths.
```
Comment (Adam-Dupaski) [inline README.md:56]: @NatasaKralj Please add correct formatting to "C:\Projects\Mendix\docs\hugo".

Comment (NatasaKralj) [reply README.md]: Updated

File: README.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ Node.js is a JavaScript runtime. You can install it by downloading the installer
 
 ##### Windows
 
-The steps in this section assume you have installed Hugo and Node in the root directory of your local repository. For example, Hugo is installed in C:\Projects\Mendix\docs\hugo. If this is not the case, you will have to edit the commands in step 3 to reflect their install paths.
+The steps in this section assume you have installed Hugo and Node in the root directory of your local repository. For example, Hugo is installed in `C:\Projects\Mendix\docs\hugo`. If this is not the case, you will have to edit the commands in step 3 to reflect their install paths.
```
Comment (Adam-Dupaski) [inline README.md:56]: @NatasaKralj There's a better choice, please see the SG.

Comment (NatasaKralj) [reply README.md]: Indeed, my bad. Thank you for the correction.

**Review by Adam-Dupaski — APPROVED** [review]

**Pull request merged**